app/javascript/readingList/__tests__/readingList.test.jsx

```diff
@@ -53,7 +53,6 @@ describe('<ReadingList />', () => {
   afterEach(() => {
     delete global.fetch;
     delete global.getCsrfToken;
-    delete global.filterXSS;
```

Comment:
With the changes made in the previous PR #11579, this is no longer required and probably never was.